modules/crud/schema.go

```diff
@@ -132,7 +137,116 @@ func getFieldType(fieldType ast.Type, fieldTypeStuct *SchemaFieldType, doc *ast.
 	return nil
 }
 
+func (m *Module) schemaValidator(dbType, col string, fields map[string]interface{}) error {
+
+	collectionFields := m.schema[dbType][col]
+	for fieldKey, fieldValue := range collectionFields {
+		// TODO: check if internal elements of list is required for this we need to change schmea parser
```

Comment:
Is something still pending?

Reply:
if the schema is defined as [String!]!
we only store information about [] list, if it is nullable or not
we do not store information about String if it is nullable or not

To implement the above we need to make some changes in parseSchema Function

Reply:
Do we need that information?